### DISCUSSION TITLE
Azure Optimization Engine and multi tenancy

### DISCUSSION THREAD

**Discussion by ChristianLloydGrannes** [microsoft/finops-toolkit#1006] [Q&A] [answered]
I have deployed the AOE in a multi tenancy setup as described in the guide. However we are using different Log Analytics workspaces for that tenant. Since you only can define workspace ID to add additional ones, how will this work with two tenants? Today we have two LAW for each tenant, prod and non-prod servers. 

https://microsoft.github.io/finops-toolkit/optimization-engine/customize#-widen-the-engine-scope

**Comment by helderpinto** [comment, marked as answer]
Hi, @ChristianLloydGrannes. 

Thanks for the question. The AOE-specific data (all `AzureOptimization*` LAW tables) is centralized in the main tenant. Using `Perf` table data from LA workspaces in other tenants (via the `AzureOptimization_RightSizeAdditionalPerfWorkspaces` variable) is not supported at the moment. Please, raise a feature request issue if this is important for your organization.
